src/Console/Command/FixCommand.php

```diff
@@ -169,12 +169,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
             }
 
             $configFile = $resolver->getConfigFile();
-            $stdErr->writeln(sprintf('Loaded config <comment>%s</comment>%s.', $resolver->getConfig()->getName(), null === $configFile ? '' : ' from "'.$configFile.'"'));
+            $stdErr->writeln(
+                sprintf(
+                    'Loaded config <comment>%s</comment>%s.',
+                    $resolver->getConfig()->getName(),
+                    null === $configFile ? '' : ' from "'.$configFile.'"'
+                ),
+                OutputInterface::VERBOSITY_VERBOSE
```

Comment:
please no. we had wasted so many time because user was not aware that he is using a config file